_posts/en/newsletters/2018-07-24-newsletter.md

```diff
@@ -0,0 +1,169 @@
+---
+title: 'Bitcoin Optech Newsletter #5'
+permalink: /en/newsletters/2018/07/24/
+name: 2018-07-24-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+version: 1
+---
+This week's newsletter includes news about a new method under
```

Comment:
I think "**news** about a **new**..." sounds a bit clumsy. I've reworded in my fixup commit.

Reply:
What about "This new week's new newsletter includes new news about a new method"?  :-)  (I'm going with the version in your commit.)